psasim/test/run_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
Can we add a header on this file?

    # Copyright The Mbed TLS Contributors
    # SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later

And or even a help line to assist people running it for the first time?